basic-features.md

```diff
@@ -38,12 +44,16 @@ The following table provides an overview of the feature development history of T
 | [Unique indexes](/constraints.md)                                                                        | Y            | Y            | Y            | Y            |
 | [Clustered index on integer `PRIMARY KEY`](/constraints.md)                                              | Y            | Y            | Y            | Y            |
 | [Clustered index on composite or non-integer key](/constraints.md)                                       | Y            | Y            | Y            | N            |
+
+## SQL statements
+
 | **SQL statements** [^2]                                                                                  | **5.2**      | **5.1**      | **5.0**      | **4.0**      |
+|----------------------------------------------------------------------------------------------------------|:------------:|:------------:|:------------:|:------------:|
 | Basic `SELECT`, `INSERT`, `UPDATE`, `DELETE`, `REPLACE`                                                  | Y            | Y            | Y            | Y            |
 | `INSERT ON DUPLICATE KEY UPDATE`                                                                         | Y            | Y            | Y            | Y            |
 | `LOAD DATA INFILE`                                                                                       | Y            | Y            | Y            | Y            |
 | `SELECT INTO OUTFILE`                                                                                    | Y            | Y            | Y            | Y            |
-| `INNER JOIN`, `LEFT\|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
+| `INNER JOIN`, `LEFT|RIGHT [OUTER] JOIN`                                                                 | Y            | Y            | Y            | Y            |
```

Comment:
This is causing display issues in GitHub markdown. Hopefully it is not an issue in the live markdown.

Reply:
Thanks! I change it back.